src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/AttachedPageBase.java

```diff
@@ -0,0 +1,40 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.logging.Log;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.nio.charset.StandardCharsets;
+import org.openqa.selenium.WebDriverException;
+
+public abstract class AttachedPageBase<T extends AttachedPageBase<T>> extends BasePageObject{
+
+	public String siteSpecificPath;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make siteSpecificPath a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)